<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="prompt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the name of a course you want to register </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Included</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addCourse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>63</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listCourses">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCourses_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>231</width>
      <height>119</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>711</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="TableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>591</height>
       </rect>
      </property>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>9:00-9:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9:30-10:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10:00-10:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10:30-11:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11:00-11:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11:30-12:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12:00-12:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12:30-13:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13:00-13:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13:30-14:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14:00-14:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14:30-15:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15:00-15:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15:30-16:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16:00-16:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16:30-17:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17:00-17:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17:30-18:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18:00-18:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18:30:19:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19:00-19:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19:30-20:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20:00-20:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20:30-21:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21:00-21:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21:30-22:00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22:00-22:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22:30-23:00</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Mon</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tue</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thu</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fri</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sat</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>580</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Timetable</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCredit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove all courses</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total credits: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="proceedButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inputed course(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timetable information</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show dialog</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>e.g COMP 2012H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Semester: </string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTerm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fall 2019</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Note: TBA-time classes will not be considered</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
